week2/homework/src/todo.js

```diff
@@ -0,0 +1,137 @@
+const fs = require('fs');
+
+const help = () => {
+  try {
+    console.log('Command - descriptions');
+    console.log('---------------------');
+    console.log(
+      '1. list - Shows current to-dos, or shows an appropriate text if there are no to-dos'
+    );
+    console.log('---------------------');
+    console.log(
+      '2. add - Adds a to-do item. All the words behind add are entered as 1 to-do item to the list.'
+    );
+    console.log('---------------------');
+    console.log('3. remove - Removes a to-do item by its 1-base index, e.g. to remove second item');
+    console.log('---------------------');
+    console.log('4. reset - Removes all to-do items from the list');
+    console.log('---------------------');
+    console.log('5. update - Updates a to-do item with new text');
+  }
+ catch (error) {
+    return console.log(error.message);
+  }
+};
+
+const fetchData = () => {
+  try {
+    const dataString = fs.readFileSync('todoData.json');
+    return JSON.parse(dataString);
+  }
+ catch (error) {
+    return [];
```

Comment:
😞 
If you are going to ignore an error add a comment to explain to other developers why

Reply:
I have read an article about rethrow errors, I have just use it in my homework to know does it useful or not.